osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/config/CoreNotificationClientConfig.java

```diff
@@ -32,6 +32,9 @@
 @PropertySource(value = "file:${osgp/AdapterWsCore/config}", ignoreResourceNotFound = true)
 public class CoreNotificationClientConfig extends AbstractConfig {
 
+  private static final String PROPERTY_NAME_NOTIFICATION_APPLICATION_NAME =
+      "web.service.notification.application.name";
+
```

Comment:
I think it is fine to use `@Value` here (in a config class in the application.config package):
```suggestion
  @Value("${web.service.notification.application.name}")
  private String applicationName;
```